src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -85,6 +85,7 @@ def load_arguments(self, _):
         c.argument('enable_pod_security_policy', action='store_true')
         c.argument('disable_pod_security_policy', action='store_true')
         c.argument('enable_acr')
+        c.argument('disable_acr')
```

Comment:
couldn't we use "--enable-acr false" instead?

Reply:
this keeps the same pattern as other options above.